aspnetcore/blazor/class-libraries.md

```diff
@@ -106,6 +106,42 @@ Welcome to your new app.
 
 Include the `@using MyComponentLib1` directive in the top-level *_Import.razor* file to make the library's components available to an entire project. Add the directive to an *_Import.razor* file at any level to apply the namespace to a single page or set of pages within a folder.
 
+## Create a Razor components class library with static assets
+
+An RCL can include static assets. The static assets are available to any app that consumes the library. For more information, see <xref:razor-pages/ui-class#create-an-rcl-with-static-assets>.
+
+## Add an XML linker configuration file to a library
+
+Embed the XML file into the library as an embedded resource.
+
+For example, create a *LinkerConfig.xml* file in the library that targets the `System.Linq.Queryable` type in the `System.Core` assembly:
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<linker>
+  <assembly fullname="System.Core">
+    <type fullname="System.Linq.Queryable" preserve="all" />
+  </assembly>
+</linker>
+```
+
+For information on the IL linker format, see [Link xml file examples (mono/linker GitHub repository)](https://github.com/mono/linker#link-xml-file-examples).
+
+The library's project file has:
+
+* A package reference for [System.Linq.Dynamic.Core](https://www.nuget.org/packages/System.Linq.Dynamic.Core/).
```

Comment:
Actually, can we not use System.Linq.Dyanmic.Core as an example? The community is working with the owner of that library to include the XML linker config in that library, so that consumers of the library no longer need to.